styles.css

```diff
@@ -427,3 +429,123 @@ form {
   color: #42526e;
   margin-bottom: 300px;
 }
+
+.desktop-menu {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #172b4d;
+}
+
+.desktop-menu-list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  gap: 30px;
+}
+
+@media only screen and (min-width: 768px) {
+  #welcome {
+    display: none;
+  }
+
+  #nav {
+    display: none;
+  }
+
+  #background {
+    background-size: cover;
+    background-image: url("header-background-desktop.svg");
+    width: 100vw;
+    height: 100vh;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+  }
+
+  #description {
+    width: 50vw;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    flex-wrap: wrap;
+  }
+
+  .card-2-container {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    flex-wrap: wrap;
+  }
+
+  .about-me-container {
+    background-image: url("about-right-image.png"), url("about-left-image.png");
+    background-repeat: no-repeat, no-repeat;
+    background-position: right, left;
+  }
+
+  .about-me-description {
+    width: 50vw;
+  }
+
+  .contact-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    flex-direction: row;
+    background-image: url("contact-left-image.png");
+    background-repeat: no-repeat;
+    background-position: left;
+  }
```

Comment:
- [ ] Hey, great job with your design, however, let's kindly make a few adjustments to make our design better compared to the Figma design in the images below:
   - Let's align the form content to be on the same line with the card's content
   - Let's align the form button correctly as seen in the Figma design.
   - Let's improve the text also to be on the same line as the card and the text length as that in Figma. YOu can tweak the text-align, and play around with it to see what best fix as seen in the Figma design.




*your design*
![Screenshot from 2023-01-27 22-21-54](https://user-images.githubusercontent.com/59397159/215203818-ce6fa42c-06c8-40dc-a1b7-5f4f4302064c.png)

*Figma design*
![Screenshot from 2023-01-27 22-20-49](https://user-images.githubusercontent.com/59397159/215203829-e9bdb22b-f1b1-47bd-a89a-8524b8acc4ca.png)

Reply:
Fixed.